tools/clang/test/HLSLFileCheck/hlsl/semantics/legacy/resource_in_entry_point_ps.hlsl

```diff
@@ -0,0 +1,34 @@
+// RUN: %dxc /E main -T ps_6_0 %s | FileCheck %s
+// CHECK: warning: effect state block ignored - effect syntax is deprecated. To use braces as an initializer use them with equal signs. [-Weffects-syntax]
```

Comment:
This test contains a bunch of unrelated code.  The root sig is unnecessary, the SamplerState declaration is unused, the global texture declaration is unused and may only confuse someone reading the test, since the name is the same as the input resource in main(), and the main_vertex entry point is unused.

It seems that the test could be reduced to this:
```
int main(float2 texCoord : TEXCOORD0, Texture2D<float> tex  : TEXUNIT0) : SV_Target {
  return 3;
}
```

Your next test also tests SamplerState, which should also be tested without explicit `uniform` attribute, and fail in the same way.  Though perhaps the error message would need to be updated to make more sense with SamplerState as well.